docs/getting-started/get-copy.rst

```diff
@@ -0,0 +1,57 @@
+Working with PyKOALA and git
+=============================
+
+Getting a copy
+--------------
+
+To get PyKOALA, clone the repository, using git:
+
+::
+    
+    git clone https://github.com/pykoala/koala.git
+
+
+Alternatively, you can create your own :doc:`fork<../developer-guide/fork>` if you want to contribute to the code. We describe the steps for developers in this :doc:`guide<../developer-guide/index>`.
+
```

Comment:
I guess the following pertains more to developers. Users just need to install and run the code (e.g. the tutorials)

Reply:
I removed the reference to the developer guide so the content for users and developers is clearly separated.

Reply:
@paranoya can you review this change?

Reply:
Perfect :^)